packages/react-components/react-storybook-addon-export-to-sandbox/src/sandbox-factory.ts

```diff
@@ -35,6 +35,14 @@ function addActionButton(container: HTMLElement, config: Data, classList: string
   const files = scaffold[config.bundler](config);
   const action = actionConfig[config.provider];
 
+  // Check if button does not exist already in container
+  for (let i = 0; i < container.children.length; i++) {
+    const child = container.children[i];
+    if (child.tagName === 'BUTTON' && child.textContent === 'Open in CodeSandbox') {
```

Comment:
Will this work with the StackBlitz option? it does seem like we test for it: https://github.com/microsoft/fluentui/blob/3f77b502c9e3d91a27ddf6da48ff2ab9b666b91c/packages/react-components/react-storybook-addon-export-to-sandbox/src/sandbox-factory.spec.ts#L98

Reply:
Oops yeah, checking for `action.label` instead now. Great catch!